assets/sass/patterns/molecules/site-header-nav-bar-primary.scss

```diff
@@ -51,10 +48,34 @@
   text-decoration: none;
 }
 
+.nav-primary__item--list {
+  @include margin(-35, "left");
+}
+
 .nav-primary__item--search {
   float: right;
   margin-left: auto;
-  @include padding(8, "right");
+  a {
+    background-image: url("../../assets/img/patterns/molecules/nav-primary-search-ic_1x.png");
```

Comment:
The logic for that background image is repeated three times, please, create a mixin and reuse the code

Reply:
I changed the background image logic and created a mixin